hwbench/environment/block_devices.py

```diff
@@ -0,0 +1,100 @@
+import pathlib
+from json import loads
+
+import pyudev
+
+from hwbench.utils.external import External
+
+
+class Block_Device:
+    """Block_Device is a class that gathers block_device SMART data using smartctl"""
```

Comment:
"gathers block_device SMART data using smartctl"
Not only, the description seems not matching what the class is doing

Reply:
changed it since better ?